frontend/www/js/omegaup/arena/location.ts

```diff
@@ -25,8 +25,9 @@ export function getOptionsFromLocation(location: string): LocationOptions {
   // Location string is of the forms:
   // - `#problems/${alias}`
   // - `#problems/${alias}/new-run`
-  // - `#problems/${alias}/show-run:xyz`
+  // - `#problems/${alias}/show-run:${guid}`
   // - `#clarifications/${alias}/new`
+  // - `#runs/show-run:${alias}/`
```

Comment:
```suggestion
  // - `#runs/show-run:${guid}`
```

no debería ser `guid` según L65? también, esto no termina con `/`.

Reply:
Listo